### DISCUSSION TITLE
Add a previous lesson button for easier navigation

### DISCUSSION THREAD

**Discussion by fadingmorsecode** [TheOdinProject/theodinproject#4303] [Ideas]
Often times I'd like to go back to a previous lesson without having to view the course list and that's not possible currently. Clicking on view course takes you to the top of the course list so it's just more scrolling to get to a desired lesson

**Comment by rlmoser99** [comment]
Thanks for adding a suggestion. However, this is already an issue, therefore I am closing this duplicate.
https://github.com/TheOdinProject/theodinproject/issues/2623
